File: src/main/java/com/back/together02be/users/dto/request/LoginReq.java
package com.back.together02be.users.dto.request;

import jakarta.validation.constraints.NotBlank;

public record LoginReq(
        @NotBlank(message = "아이디를 입력해주세요.")
        String username,

        @NotBlank(message = "비밀번호를 입력해주세요.")
        String password
) {}

File: src/main/java/com/back/together02be/users/dto/request/LoginReq.kt
package com.back.together02be.users.dto.request

import jakarta.validation.constraints.NotBlank

data class LoginReq(
    @field:NotBlank(message = "아이디를 입력해주세요.")
    val username: String,

    @field: NotBlank(message = "비밀번호를 입력해주세요.")
    val password: String
)
